Randolph-dev/Druid_Partnering_Project_React24k
# Scrum 139 paragraph content for jobspage

## Related ticket:
[SCRUM-139](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-139?atlOrigin=eyJpIjoiM2RlMjMwOGNmNzA1NDY0M2E5Nzk5OGI3N2FjYjQxNzAiLCJwIjoiaiJ9)

## In this PR:
- Updated db with new content for Jobs page sections
- FAQ, Jobs, Team members sections
- Adjustments to the layout

## Testing instruction
- Checkout branch
- Import the latest DB and Config
- Check the jobspage 

[SCRUM-139](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-139?atlOrigin=eyJpIjoiM2RlMjMwOGNmNzA1NDY0M2E5Nzk5OGI3N2FjYjQxNzAiLCJwIjoiaiJ9)

[SCRUM-139]: https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-139?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SCRUM-139]: https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-139?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
